controllers/storagecluster/storageclasses.go

```diff
@@ -314,6 +316,9 @@ func newCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster) S
 	if initData.Spec.ManagedResources.CephBlockPools.DefaultStorageClass {
 		scc.storageClass.Annotations[defaultStorageClassAnnotation] = "true"
 	}
+	util.If(!util.IsAnnotationTruthy(initData, keyRotationEnableAnnotation), func() {
+		util.AddAnnotation(scc.storageClass, keyRotationEnableAnnotation, "false")
+	})
 	return scc
```

Comment:
Don't we need to issue an update comment for the storagecluster to apply the new annotation?

Reply:
`newCephBlockPoolStorageClassConfiguration` generates config for a new SC that is going to be created. Upon creation, it will have the annotation present.